include/retdec/common/function.h

```diff
@@ -53,6 +54,7 @@ class Function : public retdec::common::AddressRange
 
```

Comment:
_(A question not related to the attached line.)_

> When user specifies that the return type is integer, the decompiler should not continue to do heuristics to detect return type.

This will be implemented in a separate PR, right?

Reply:
Yes it will be. I have this implemented on separate branch `config-user-info`. I wanted this PR to provide a new function type only.